components/Forms/Radio.tsx

```diff
@@ -33,7 +33,7 @@ export const Radio: React.VFC<InputProps> = observer((props) => {
 
   return (
     <div className="radio">
-      <div className="mb-2.5">
+      <div className="">
```

Comment:
do we need to keep this div

Reply:
Yes, otherwise the "more information" link will come to the end of this line.